core/model/messages/PullFeedValuesMessage.h

```diff
@@ -14,23 +14,18 @@
  * limitations under the License.
  */
 
-#ifndef FILEUPLOADABORT_H
-#define FILEUPLOADABORT_H
+#ifndef WOLKABOUTCORE_PULLFEEDVALUESMESSAGE_H
+#define WOLKABOUTCORE_PULLFEEDVALUESMESSAGE_H
 
-#include <string>
+#include "core/model/messages/MessageObject.h"
 
 namespace wolkabout
 {
-class FileUploadAbort
+class PullFeedValuesMessage : public MessageObject
```

Comment:
Does this message exist in protocol?

Reply:
Yes, this is the message sent out as `d2p/{deviceKey}/pull_feed_values`.